src/lib/Scenes/Show/Components/ShowHeader/ShowHeader.tsx

```diff
@@ -33,7 +38,7 @@ export class ShowHeader extends React.Component<Props> {
         <Serif size="3" textAlign="center">
           {exhibition_period}
         </Serif>
-        <Box px={2}>
```

Comment:
`<Box px={2} mb={2}>` would accomplish the same thing `mb = marginBottom`

Reply:
Ah ok thanks!